Platformio/HAL/Targets/ESP32/display/display.cpp

```diff
@@ -23,11 +24,12 @@ Display::Display(int backlight_pin, int enable_pin): DisplayAbstract(),
     pinMode(mBacklightPin, OUTPUT);
     digitalWrite(mBacklightPin, HIGH);
 
-    ledcSetup(LCD_BACKLIGHT_LEDC_CHANNEL, LCD_BACKLIGHT_LEDC_FREQUENCY, LCD_BACKLIGHT_LEDC_BIT_RESOLUTION);
-    ledcAttachPin(mBacklightPin, LCD_BACKLIGHT_LEDC_CHANNEL);
-    ledcWrite(LCD_BACKLIGHT_LEDC_CHANNEL, 0);
+    setupBacklight(); // This eliminates the flash of the backlight
 
-    setupTFT();
+    // This backlight init causes the backlight to come on full during startup
```

Comment:
I guess since we swapped over to IDFs version of ledc we can probably just delete this.

Reply:
Yea, I don't think anyone would want to go back to it unless someone figures out a way to make the inverted case work right. I'll remove it.